src/glass/src/app/pages/services-page/services-page.component.ts

```diff
@@ -151,6 +158,12 @@ export class ServicesPageComponent {
                 this.showMountCmdDialog(cmdArgs);
               });
             }
+          },
+          {
+            title: TEXT('Remove service'),
```

Comment:
We need to decide if we want to name it `delete` or `remove`. Any preferences?

Reply:
I'm personally in favor of `remove`. But I can't tell the reason. It's more of a "feeling". My "feeling" might be wrong, e.g. we're already using `delete` at some places.